### PR TITLE
Bug fixes

### LINKED ISSUE #4878: Rule throws and error when trying to create a new rule
### Describe the bug
When creating a new rule or attempting to update an existing rule with a DB version higher than 5.x of MariaDB or MySQL. 

write a concise description of what the bug is.
https://github.com/openemr/openemr/pull/2928

### To Reproduce

image

1. Select Administration
2. Select practice
3. Select rules
4. Select any rule

### Expected behavior

Work correctly to enter a rule. 

### Client configuration

- Browser:
- OpenEMR version: 6.0.0 (3)
- Operating system: Ubuntu

### LINKED ISSUE #4879: Procedure Configuration Escaping
### Describe the bug
This has bugged me(forgive the pun) for a few months. Started in v6 worked in v5.0.2.
Found because this order `FISH, BURKITT'S/NHL/ALL` wasn't being escaped correctly with text()(would work if echoed direct to html). Failed to escape `'` single quote;

Problem is that the configuration is rendered with ajax call returning a jquery.html() so entire result should be escaped with a js_escape()(json_encode()).

### PR DESCRIPTION
**Pull request by sjpadgett**
consolidated php rendered html and escaped.

Fixes #4879
also Fixes #4878 
#### Short description of what this resolves:
Fix procedure configuration escaping.
see issue

### CODE REVIEW

**Review by juggernautsei — COMMENTED** [review]
Jerry, when I implemented this change. 
The view only dumps the SQL out to the screen without rendering the view. Trying to track down the view.

File: interface/orders/types_ajax.php [hunk 1]
```diff
@@ -121,18 +123,21 @@
...
-    echo "</tr>";
+    $render .= "</td>";
+    $render .= "</tr>";
 }
 
-echo "</table>');\n";
-echo "nextOpen();\n";
+$render .= "</table>'\n";
+$rendered = "$('#con" . attr($id) . "').html(";
+$rendered .= js_escape($render) . ");";
```
Comment (bradymiller) [inline interface/orders/types_ajax.php:141]: 2DV-2

**Comment by sjpadgett** [comment]
> I can't delete this is not an issue it fixed

@juggernautsei The functionality works for me on master. Although appears to be some over escaping causing html to appear in inputs etc.
I don't want to get started fixing CDR as it would require several days.
My concern is the deprecated sql function where the best I can tell works with the SHA1().

**Comment by sjpadgett** [comment]
Actually CDR is pretty nicely done from structural POV and the age of project. Styling needs to catch up. Completely don't understand why bootstrap was specifically excluded in header class! Looks like we have jquery-ui and jquery libraries included in project as separate dependencies directory. Suppose I could maybe do a little work on project. If anything, for peace of mind:)

**Comment by juggernautsei** [comment]
One day I will be able to flow in and out of code as you do.

**Comment by sjpadgett** [comment]
What, trying to take my job there Sherwin.:)

**Comment by juggernautsei** [comment]
Some day. I can dream. It has been said if you don't aim for something. You hit nothing.

**Comment by sjpadgett** [comment]
check it out
image

image

**Comment by juggernautsei** [comment]
That looks better. Did you have to add the container div?

**Comment by sjpadgett** [comment]
actually container-xl so goes 100% width after 1220px breakpoint.

**Comment by sjpadgett** [comment]
bringing in and will do separate PR for rules UI

**Pull request merged**